### PR TITLE
Size t dtoh

### PR DESCRIPTION
**Pull request by UplinkCoder**
should treat `size_t` as a special alias and emit it

### CODE REVIEW

**Comment by thewilsonator** [comment]
This needs a test.

**Comment by UplinkCoder** [comment]
Ah right my bad forgot to add it.

Am Mi., 4. März 2020 um 10:00 Uhr schrieb Nicholas Wilson <
notifications@github.com>:

**Comment by UplinkCoder** [comment]
@thewilsonator done!

**Review by thewilsonator — APPROVED** [review]

File: src/dmd/dtoh.d [hunk 1]
```diff
@@ -726,8 +729,17 @@ public:
                 visit(cast(AST.Dsymbol)ad);
                 return;
             }
+
+            // for function pointers we need to original type
```
Comment (ibuclaw) [inline src/dmd/dtoh.d]: Sentence seems (incomplete), or missing a bit (in the) middle.

File: src/dmd/dtoh.d [hunk 2]
```diff
@@ -1667,7 +1682,12 @@ public:
             if (i)
                 buf.writestring(", ");
             auto fparam = AST.Parameter.getNth(tf.parameterList.parameters, i);
+            if (fparam.type == AST.Type.tsize_t && originalType)
```
Comment (ibuclaw) [inline src/dmd/dtoh.d]: I suspect this comparison to size_t will need to be turned into a function.

**Pull request merged**